spec/features/search/new_flow/across_england/across_england_secondary_spec.rb

```diff
@@ -0,0 +1,142 @@
+require 'rails_helper'
+
+RSpec.feature 'Searching across England' do
+  include StubbedRequests::SubjectAreas
+  include StubbedRequests::Courses
+  include StubbedRequests::Subjects
+
+  before do
+    stub_subjects
+    stub_courses_request
+    stub_subject_areas
+  end
+
+  scenario 'Candidate applies qualifications filters on results page' do
+    given_that_the_new_search_flow_feature_flag_is_enabled
+    when_i_visit_the_start_page
+    and_i_select_the_across_england_radio_button
+    and_i_click_continue
+    then_i_should_see_the_age_groups_form
+    and_the_correct_age_group_form_page_url_and_query_params_are_present
+
+    when_i_click_back
+    then_i_should_see_the_start_page
+    and_the_across_england_radio_button_is_selected
+
+    when_i_click_continue
+    then_i_should_see_the_age_groups_form
+
+    when_i_click_continue
+    i_should_see_an_age_group_validation_error
```

Comment:
Similarly here, you've tested lines 15-30 in `spec/features/search/new_flow/across_england/across_england_primary_spec.rb` and `spec/features/search/new_flow/across_england/across_england_further_education_spec.rb` . These types of feature tests can be expensive (slow, to maintain and execute) so we would want to try and minimise these.

Reply:
👍